src/main/java/com/spirit21/swagger/converter/parsers/ParameterParser.java

```diff
@@ -132,14 +158,32 @@ private Parameter getParameter(String paramUnformatted, List<String> imports, St
                 parameter.setLocation("body");
                 parameter.setType(type);
                 parameter.setRequired(true);
+
             } else {
                 parameter.setLocation("query");
                 parameter.setName(getNameFromQueryParamAnnotation(paramUnformatted));
                 parameter.setType(type);
             }
-            parameter.setFormat(format);
-            String description = descriptionMap.get(name);
-            parameter.setDescription(description);
+            if (isDefaultValue) {
+                parameter.setLocation("query");
+                // parameter.setType(type);
+                for (int i = 0; i < defaultValue.size(); i++) {
```

Comment:
Ich bevorzuge for-Schleifen mit der Doppelpunkt-Schreibweise
`for (String value: defaultValue) {   }`